velox/functions/prestosql/types/UuidType.cpp

```diff
@@ -45,7 +45,7 @@ class UuidCastOperator : public exec::CastOperator {
       castFromString(input, context, rows, *result);
     } else {
       VELOX_UNSUPPORTED(
-          "Cast from {} to UUID not yet supported", resultType->toString());
+          "Cast from {} to UUID not yet supported", input.type()->toString());
```

Comment:
Is this related to this change ? If not, lets move it to another PR.

Reply:
Not related. Sure will open another PR.

Reply:
Opened here
https://github.com/facebookincubator/velox/pull/11105